### DISCUSSION TITLE
Cannot have access to the formula generated data if I filter on the rows on current user

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2775] [Help] [answered]
Hello,

If I use as data provider current user.myTable.Created by, I cannot have access to the formula type data. 
I can have access to them if my data provider is myTable but I cannot filter on Created by current user in the filters

What can I do? 

Thanks a lot

**Comment by mjashanks** [comment, marked as answer]
Hey Oswald!

Thanks for reporting, I have logged this as a bug: https://github.com/Budibase/budibase/issues/2796.

In the meantime, all you can do is recreate the same formula in the frontend (i.e. not a column in your data table).

You won't be able to do this in a table component, because you cannot add "Custom Columns". So, you would have to use a data provider + repeater to display your data instead: https://docs.budibase.com/building-apps/design/components/repeater. This will allow you to manually build out your rows, using sets of components. Inside each component, you can add binding expressions, just like you would do with your formula.

**Reply by oswaldB** [reply]
Hello :)

fine  for the second way tod o it but how I can filter on the rows created by the user only?

Thanks a lot :)

**Reply by mjashanks** [reply]
Not sure I understand what you are asking me.

To filter on rows created by the user, set the data providers's `data` to `Current User.MyTable.Field-CreatedBy` (as you said originally), then add a repeater in below that.

**Comment by oswaldB** [comment]
Oh sorry I understand.

I have to put a repeater in the repeater?

Thanks. I will try

**Reply by mjashanks** [reply]
No - put a repeater inside the Data Provider. 

Data Providers returns the rows, a Repeater displays its contents for each row.

**Comment by oswaldB** [comment]
Yep I succeeded.

The bug fix will make it easier to manage in the future.

And it will help with follow up data as well.

Have a great day
